src/components/_internal/BaseDateRangePicker/styles.ts

```diff
@@ -9,7 +9,7 @@ import {
   getLineHeight,
   getRadii,
   pxToRem,
-} from '../../../../utils';
+} from '../../../utils';
 
 export const datePickerStyles = css`
   .DateRangePicker-input {
```

Comment:
Could you please add `font-family: ${getFontFamily('base')};` to this `.DateRangePicker-input` class

Reply:
Also please change 
`height: ${pxToRem(32)};` -> `height: ${getFormStyle('fieldHeight')};`